### PR TITLE
Add Easy Digital Downloads integration. #351

### PR DESCRIPTION
**Pull request by mindctrl**
#351 

Leaving unmerged until we firm everything up.

### CODE REVIEW

File: includes/integrations/class-rcp-easy-digital-downloads.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+
+function rcp_edd_init() {
+
+	if ( ! class_exists( 'Easy_Digital_Downloads' ) ) {
+		return;
+	}
+	new RCP_EDD;
+}
+add_action( 'init', 'rcp_edd_init' );
```
Comment (pippinsplugins) [inline includes/integrations/class-rcp-easy-digital-downloads.php]: Some of the EDD used above are tied to `init`. Please double check the priorities to ensure using default `10` works.

Comment (mindctrl) [inline includes/integrations/class-rcp-easy-digital-downloads.php]: Everything worked in my tests, but I'll be doing some more thorough testing over the next couple days... with different extensions, different servers, etc.

File: includes/integrations/class-rcp-easy-digital-downloads.php [hunk 2]
```diff
@@ -79,6 +80,35 @@ public function edd_downloads_query( $query, $atts ) {
...
+	 * @access public
+	 * @since 2.7
+	 */
+	public function edd_downloads_excerpt( $excerpt ) {
+
+		global $rcp_options;
+
+		$post_id = get_the_ID();
+
+		if ( $this->member->can_access( $post_id ) ) {
```
Comment (pippinsplugins) [inline includes/integrations/class-rcp-easy-digital-downloads.php]: This should show the excerpt if `Show excerpt` is enabled on the product.

Comment (mindctrl) [inline includes/integrations/class-rcp-easy-digital-downloads.php]: @pippinsplugins good catch. updated.

**Pull request merged**